src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1534,6 +1540,12 @@ private static FileChange GetFileChangeDetails(Range changeRange, string insertS
             ILogger Logger,
             CancellationToken cancellationToken)
         {
+            // If filesToAnalzye is empty, nothing to do so return early.
+            if (filesToAnalyze.Length == 0)
+            {
+                return;
```

Comment:
Shouldn't this be `Task.FromResult(true);`

Reply:
Good question.  I copied line 1558 which just does a return.  I "think" that is OK for a `void` return Task - isn't it?

Reply:
Yeah, you only need `Task.FromResult` if the method isn't tagged as `async`.  When it's tagged as `async` the compiler generates the code for managing task objects for you.

In fact if you did add `Task.FromResult(true)` you'd have to await it, and then you'd have to `return` on the next line.